### PR TITLE
fix: missing translations

### LINKED ISSUE #176: Not all text are translated
**Describe the bug**
Not all text are translated in Filipino.
Text that were not translated are: 
- error "the input format is incorrect"
- error "the UIN is invalid"
- errors from video (see video/screenshots in Screenshots)
- history - time since action
- history - action (downloaded, shared, requested)
- headers home, request, profile

**To Reproduce**
1. Change language to Filipino
2. Navigate to the pages mentioned above
3. Note that not all text are translated to Filipino

**Expected behavior**
All text should be translated to Filipino

**Reproduced on**
0.3.0rc2

**Screenshots**
https://user-images.githubusercontent.com/102940764/166642366-f80faef7-9c30-44c8-97d6-18b48f28e771.mp4
Screenshot_20220428-194943_MOSIP Resident App
Screenshot_20220428-192106_MOSIP Resident App
Screenshot_20220428-192138_MOSIP Resident App

### PR DESCRIPTION
**Pull request by pmigueld**
Fixes #176

### CODE REVIEW

**Review by danicaerediano — APPROVED** [review]
LGTM

File: screens/MainLayout.tsx [hunk 1]
```diff
@@ -35,6 +38,7 @@ export const MainLayout: React.FC<RootRouteProps> = () => {
           component={route.component}
           options={{
             ...route.options,
+            title: t(route.name.toLowerCase()).toUpperCase(),
```
Comment (nicholemnl) [inline screens/MainLayout.tsx:41]: I this intended? isn't toUpperCase would still transform text to uppercase without lower case right?

Comment (pmigueld) [reply screens/MainLayout.tsx:41]: The lowercase transform is for the translation key. Then uppercase according to the design spec for the header text.

**Review by nicholemnl — APPROVED** [review]
Looks good to me @pmigueld

**Pull request merged**